charts/promtail/values.yaml

```
@@ -431,9 +439,11 @@ config:
 
     client:
       url: {{ .Values.config.lokiAddress }}
+      {{- toYaml .Values.extraClientConfigs | nindent 2 }}
 
     positions:
       filename: /run/promtail/positions.yaml
 
     scrape_configs:
       {{- tpl .Values.config.snippets.scrapeConfigs $ | nindent 2 }}
+      {{- toYaml .Values.extraScrapeConfigs | nindent 2 }}
```

As suggested and to be consistent, these should go under `config.snippets`. We may also want to make them templated. As such, they should be strings.

```suggestion
      {{- tpl .Values.config.snippets.extraClientConfigs . | nindent 2 }}

    positions:
      filename: /run/promtail/positions.yaml

    scrape_configs:
      {{- tpl .Values.config.snippets.scrapeConfigs . | nindent 2 }}
      {{- tpl .Values.config.snippets.extraScrapeConfigs . | nindent 2 }}
```